docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -552,6 +552,30 @@ To execute this update, simply run
 ./call-path.sh kyc-in-process.json
 ```
 
+##### KYC Being Collected/Verified
+
+:::tip
+
+This step is optional. Most business don't use it: you can skip it and go to the [next one](#ready-to-receive-funds)
+
+:::
+
+Update for this action is simply changing transaction status. It's a good idea to modify message too, reflecting details of the process:
```

Comment:
I would add that using this status makes the most sense when KYC verification may need to be performed asynchronously.